plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Users.js

```diff
@@ -284,10 +284,14 @@ Ext.define('NX.coreui.controller.Users', {
             { property: 'userId', value: me.getUserSearchBox().getValue() }
           ]
         },
-        callback: cb
+        callback: function(records, operation, success) {
+          if (Ext.isFunction(cb)) {
+            cb(records, operation, success);
+          }
+          me.reselect();
+        }
       });
     }
-    me.callParent();  // triggers transition between list/detail view
```

Comment:
jsdoc should then use @override, and if you aren't going to call parent, probably explain why in a comment for clarity plz